.gitignore

```diff
@@ -23,3 +23,6 @@ __pycache__/
 Taskfile.yml
 Taskfile.yaml
 /tools/deploy/environment.properties
+
+# Special file placed for gateway aap-dev testing
+src/aap_eda/settings/dev_hacked_settings.py
```

Comment:
If this file is only for development testing should be [ignored locally](https://git-scm.com/docs/gitignore) instead here?

Reply:
That's fine, adding it to the aap-dev instructions.